components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/client.ts

```diff
@@ -16,8 +16,14 @@
 
 const HttpClientFacade = Java.type("org.eclipse.dirigible.components.api.http.HttpClientFacade");
 
+
+interface Option {
+	params: Array<{name: string, value: string}>;
+}
+
+
 export class HttpClient {
-    get(_url, options) {
```

Comment:
By default all methods are implicitly `public`, however in order to align with the coding style please add explicitly the `public` access modifier:

```ts
public get(_url: string, options: Option): Object {
...
```